src/components/grid/GridComponent.vue

```diff
@@ -3,6 +3,7 @@
     <div class="row">
       <div class="col vld-parent">
         <table
+          v-if="grid && grid.length"
```

Comment:
multiple occurrences of grid && grid.length suggest to move to computed prop

Reply:
Seems redundant in this particular case.